src/components/DampAndMouldReports/components/ReportFrequencyBadge.js

```diff
@@ -0,0 +1,20 @@
+import cx from 'classnames'
+
+const ReportFrequencyBadge = ({ numberOfReports }) => {
+  return (
+    <div style={{ marginTop: '5px', marginLeft: '-5px' }}>
+      <span
+        className={cx([
+          'damp-and-mould-report-frequency-badge ',
+          numberOfReports === 0 ? 'new-report' : 'recurring-report',
+        ])}
+      >
+        {numberOfReports === 0
+          ? 'New report'
```

Comment:
Should it display 'New report' in this case?

I checked and it displays this even if all the reports are many months in the past